dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/internal/utils/TracerExtensionsTest.kt

```diff
@@ -28,50 +22,37 @@ import org.mockito.quality.Strictness
 )
 
 @MockitoSettings(strictness = Strictness.LENIENT)
-@ForgeConfiguration(Configurator::class)
+@ForgeConfiguration(Configurator::class, seed = 0x4f36670aL)
 class TracerExtensionsTest {
 
+    lateinit var tracer: Tracer
+
+    @BeforeEach
+    fun `set up`() {
+        tracer = Tracer.Builder().build()
+    }
+
     @AfterEach
     fun `tear down`() {
-        GlobalTracer.get().setFieldValue("isRegistered", false)
-        GlobalTracer::class.java.setStaticValue("tracer", NoopTracerFactory.create())
+        tracer.scopeManager().active()?.close()
```

Comment:
Are you sure this is doing the same thing as what I was doing there ? Basically that resets the current tracer and restores everything to the NoOpTracer, what you added here only closes the active scope but the tracer will still be the one you added, we will still have an active DDTracer there.